.github/workflows/push_image.yml

```diff
@@ -123,6 +128,26 @@ jobs:
         run: |-
           gcloud container clusters get-credentials ${{ secrets.k8s_cluster_id }} --zone ${{ secrets.k8s_cluster_zone }}
 
+      - name: Setup Node 14 for Tests
```

Comment:
this is copied from https://github.com/goodwatercap/company-tearsheet-web-client/blob/08b2e6d932a61ee89549e5ec2c6122567dcf68e7/.github/workflows/deploy.yml#L64, but I am suspicious we don't actually need it

Reply:
we should use the node version specified in `.nvmrc`, see https://github.com/goodwatercap/concierge-react/blob/main/.github/workflows/deploy.yml#L52. Although it doesn't look like we set that within tearsheet, so I'm guessing its using the latest lts (node 16)